source/extensions/filters/network/client_ssl_auth/client_ssl_auth.cc

```diff
@@ -22,6 +22,12 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace ClientSslAuth {
 
+struct RcDetailsValues {
+  // SSL certificate is not in the allowed principals list.
+  const std::string AuthDigestNoMatch = "auth_digest_no_match";
+};
+using RcDetails = ConstSingleton<RcDetailsValues>;
+
```

Comment:
May be simple `constexpr absl::string_view` would be better?

Reply:
I just followed what we are doing in other filters.

https://github.com/envoyproxy/envoy/blob/9dd8ae3c69a9917d97a886ed03e1c010dcd9b098/source/extensions/filters/http/csrf/csrf_filter.cc#L21

https://github.com/envoyproxy/envoy/blob/5963beae8842982803af1bef04fb5a2a0893c613/source/extensions/filters/http/ratelimit/ratelimit.cc#L21

https://github.com/envoyproxy/envoy/blob/88705d72f063b7c75b18b452a4bd9af406611faf/source/extensions/filters/http/ext_authz/ext_authz.cc#L16

https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/http/fault/fault_filter.cc#L30

Should we keep it consistent instead or you prefer doing it this way here?

Reply:
There's samples both ways. See `extensions/common/wasm/context.cc` `CloseStreamResponseDetails`. I think the constexpr string_view is simpler; please switch it to that.

Reply:
Changed to `constexpr absl::string_view` as suggested.